app/views/course/discussion/topics/_discussion_topic_programming_file_annotation.jbuilder

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
The comments in this file also apply to the other 2 similar jbuilder (submission question and vide)